pymtl3/passes/translator/behavioral/BehavioralTranslatorL3.py

```diff
@@ -20,6 +20,9 @@ class BehavioralTranslatorL3( BehavioralTranslatorL2 ):
   def __init__( s, top ):
     super( BehavioralTranslatorL3, s ).__init__( top )
 
+  def clear( s, tr_top ):
+    super( BehavioralTranslatorL3, s ).clear( tr_top )
```

Comment:
what is the purpose of clear here?

Reply:
The translators store metadata generated by the call to `tr.translate` in their attributes. At the beginning of each `translate()` the data structures that store these metadata are initialized/cleared by `clear()`.